Makefile

```diff
@@ -129,6 +129,7 @@ install: $(BINS) $(DATA)
 	install -D -m644 ./ygg.bash $(DESTDIR)$(DATADIR)/bash-completion/completions/$(SHORTNAME)
 	install -D -m644 ./yggd.bash $(DESTDIR)$(DATADIR)/bash-completion/completions/$(SHORTNAME)d
 	install -D -m644 ./data/pkgconfig/yggdrasil.pc $(DESTDIR)$(PREFIX)/share/pkgconfig/$(LONGNAME).pc
+	mkdir $(DESTDIR)$(LIBEXECDIR)/$(LONGNAME)
```

Comment:
Use `install -d` rather than `mkdir`. If for some terrifying reason `$(DESTDIR)` or `$(LIBEXECDIR)` or `$(LONGNAME)` don't exist by the time this `mkdir` is invoked, it will error. `install -d` will create the specified directory, and all components leading up to it.

Reply:
fixed (I guess that was me being extra lazy)